### PR TITLE
Script for reprodusing issue 1908

### PR DESCRIPTION
**Pull request by HSavynetska**
Script for reprodusing issue [Non-media app must be activated during active audio source](https://github.com/smartdevicelink/sdl_core/issues/1908)

This PR is **not ready** for review.
This is **draft** version of the script.

### CODE REVIEW

File: test_scripts/Defects/4_6/1908_Non_media_app_must_be_activated_during_active_audio_source.lua [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	self.hmiConnection:SendNotification("BasicCommunication.OnEventChanged", {eventName = "AUDIO_SOURCE", isActive = true})
+  self.mobileSession1:ExpectNotification("OnHMIStatus",
+  { hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" },
+  { hmiLevel = "FULL", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+  :Times(2)
+  :Do(function(exp)
+    if exp.occurences == 1 then
+      local requestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(1) })
+       EXPECT_HMIRESPONSE(requestId)
+      self.hmiConnection:SendNotification("BasicCommunication.OnEventChanged", {eventName = "AUDIO_SOURCE", isActive = false})
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1908_Non_media_app_must_be_activated_during_active_audio_source.lua]: By activation non-media app HMI should not stop music playing, so BasicCommunication.OnEventChanged(isActive = false) is redundant, please remove.

File: test_scripts/Defects/4_6/1908_Non_media_app_must_be_activated_during_active_audio_source.lua [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    end
+  end)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("RAI", common.rai_n)
+runner.Step("Activate App", common.activate_app)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1908_Non_media_app_must_be_activated_during_active_audio_source.lua]: Function common.activate_app is written for media app activation, AUDIBLE state is expected.
Please redefine activation function locally with expectation of NOT_AUDIBLE state.

File: test_scripts/Defects/4_6/1908_Non_media_app_must_be_activated_during_active_audio_source.lua [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+runner.Title("Test")
+runner.Step("onEventChange EMBEDDED_NAVI true", onEventChange)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/4_6/1908_Non_media_app_must_be_activated_during_active_audio_source.lua]: Please update test description according to sequence in test, something like "App activation by embedded navigation is active"

**Comment by GetmanetsIrina** [comment]
PR is rebased on current develop(8b2019d) branch.
Script is applicable for all policy flows and transport types.
Script has been moved to 6_1 folder.
Test set for script is added.

File: test_scripts/Defects/6_1/1908_Non_media_app_must_be_activated_during_active_audio_source.lua [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("RAI", common.registerApp)
+runner.Step("Activate App FULL", activateApp)
+runner.Step("Deactivate App BACKGROUND", deactivateApp)
+
+runner.Title("Test")
+runner.Step("AUDIO sourse is activated", onEventChange)
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1908_Non_media_app_must_be_activated_during_active_audio_source.lua:56]:
```suggestion
runner.Step("AUDIO source is activated", onEventChange)
```

**Comment by jacobkeeler** [comment]
Closing in favor of #2452

**Pull request closed without merging**